### PR TITLE
8 Puzzle Problem using A* Algorithm

### PR DESCRIPTION
**Pull request by Manasi2001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

8 Puzzle Problem Solving using A* Algorithm has been successfully done. Three initial states have been taken into account and the search technique is applied to reach the goal state.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This PR is with reference to issue #7177.

### CODE REVIEW

File: Python/search/8_Puzzle_Problem_using_A_star_Algorithm.py [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+[2, 8, 1]
+[4, 6, 3]
+[0, 7, 5] 
+
+A*: ['D', 'R', 'U', 'L', 'L', 'D', 'R', 'R', 'U', 'L', 'U', 'R']
+space: 4817
+time: 0.14372038841247559
+
+------------------------------------------
+
```
Comment (ankitaggarwal23) [inline Python/search/8_Puzzle_Problem_using_A_star_Algorithm.py]: Remove extra test case

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by Manasi2001** [comment]
You may check this again now @ankitaggarwal23.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
- Add Time and Space Complexity at the end of the Code

**Comment by Manasi2001** [comment]
> * Add Time and Space Complexity at the end of the Code

Done.

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by ankitaggarwal23** [comment]
@all-contributors bot please add @Manasi2001 for code

**Pull request merged**